interactive-calculator/src/Utils/compound-interest.js

```diff
@@ -15,11 +15,28 @@ export const calcCompoundInterest = (
   return finalSavings / 100;
 };
 
-// export const mapSavingsToObject = (label,principalAmount,
-//   interestRatePercent,
-//   timesCompoundedPerPeriod,
-//   numberOfPeriods ) => {
+export const mapSavingsToObject = (
+  label,
+  principalAmount,
+  interestRatePercent,
+  timesCompoundedPerPeriod,
+  numberOfPeriods
+) => {
+  const num = 10;
+  let arrayData = Array(num);
```

Comment:
This seems redundant considering the next few lines of code that reassign the array to the return from .map()